src/components/Product/ProductInsert.jsx

```diff
@@ -104,39 +104,39 @@ class ProductInsert extends Component {
                     label="Product Name"
                     name="prodName"
                     type="text"
-                    dataVal={prodName}
+                    data={prodName}
                     handleChange={this.handleChange}
                   />
 
                   <InputBox
                     label="Product ID Code"
                     name="code"
                     type="text"
-                    dataVal={code}
+                    data={code}
                     handleChange={this.handleChange}
                   />
 
                   <InputBox
                     label="Description"
                     name="descript"
                     type="text"
-                    dataVal={descript}
+                    data={descript}
                     handleChange={this.handleChange}
                   />
 
                   <InputBox
                     label="Price in $"
                     name="price"
                     type="number"
-                    dataVal={price}
+                    data={price}
                     handleChange={this.handleChange}
                   />
 
                   <InputBox
                     label="Does this product have an expiration date? Enter 1 for Yes, and enter 0 for No"
                     name="expirable"
                     type="text"
-                    dataVal={expirable}
+                    data={expirable}
```

Comment:
Probably needs to be `value`

Reply:
Done

Reply:
To remove user, I changed one of user's first name and last name to NULL, email and password to ' '. If I need change it back to original we have in file, let me know. Thank you!